From 982d5397d7c6d5c70bdb868a5232c7df785c65f2 Mon Sep 17 00:00:00 2001
From: "Wu, Hao" <hao.wu@intel.com>
Date: Wed, 17 Dec 2014 14:16:07 +0800
Subject: [PATCH 1/2] extcon-3gpio_otg: use fs_initcall instead of
 subsys_initcall

In cherrytrail, gpio driver is using fs_initcall not
subsys_initcall, so loaded later than extcon 3gpio
driver. Move it to fs_initcall instead to avoid the
driver loading order issue.

Change-Id: I62a9dc767a9856c9ea0aceb62139703825da2bd8
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-13516
Signed-off-by: Wu, Hao <hao.wu@intel.com>
---
 drivers/extcon/extcon-3gpio_otg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/extcon/extcon-3gpio_otg.c b/drivers/extcon/extcon-3gpio_otg.c
index 664faef..4de27ed 100644
--- a/drivers/extcon/extcon-3gpio_otg.c
+++ b/drivers/extcon/extcon-3gpio_otg.c
@@ -236,7 +236,7 @@ static int __init vuport_init(void)
 {
 	return platform_driver_register(&vuport_driver);
 }
-subsys_initcall(vuport_init);
+fs_initcall(vuport_init);
 
 static void __exit vuport_exit(void)
 {
-- 
1.9.1

